src/Hint/Pragma.hs

```diff
@@ -52,7 +53,7 @@ optToPragma :: [(Located AnnotationComment, [String])]
              -> [(Located AnnotationComment, [String])]
              -> [Idea]
 optToPragma flags langExts =
-  [pragmaIdea (OptionsToComment (map fst old) ys rs) | old /= []]
+  [pragmaIdea (OptionsToComment (fst <$> o NE.:| os) ys rs) | o:os <- [old]]
```

Comment:
```suggestion
  [pragmaIdea (OptionsToComment (fst <$> old2) ys rs) | old2 <- [NE.nonEmpty old]]
```

Is that a neater formulation?

Reply:
Wouldn't you'd need `| Just old2 <-` instead of `| old2 <-` for that to be valid? Given that, do you still consider it neater? If so, I'm happy to change to it.

Reply:
Yep, you would need to do that. I would consider it neater (no chance of accidentally taking apart the list and putting it back in the wrong order)